SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -6,33 +6,57 @@
 using System.Configuration;
 using System.IO;
 using System.Reflection;
+using System.Xml;
 
 namespace Microsoft.AppCenter.Utils
 {
     public class DefaultApplicationSettings : IApplicationSettings
     {
         private const string FileName = "AppCenter.config";
+        private const string BackupFileName = "AppCenter.config.bak";
+        private const string CorruptedConfigurationWarning = "Configuration is corrupted. App Center could work incorrectly.";
         private static readonly object configLock = new object();
         private static Configuration configuration;
 
         internal static string FilePath { get; private set; }
 
+        internal static string BackupFilePath { get; private set; }
+
         public DefaultApplicationSettings()
         {
             lock (configLock)
             {
-                configuration = OpenConfiguration();
+                try
+                {
+                    configuration = OpenConfiguration();
+                    CreateConfigurationFileBackup();
```

Comment:
Why are we creating a backup after opening?
If the goal is just to avoid a crash we can just delete and start over like we do for sqlite.